web/src/components/Footer.tsx

```diff
@@ -12,18 +12,14 @@ const FooterContainer = styled.footer<{ overlay: boolean }>`
   display: flex;
   flex-wrap: wrap;
   justify-content: center;
-  padding: ${props => (props.overlay ? `0 0 0 10px` : `15px 5px`)};
+  padding: ${props => (props.overlay ? '0 0 0 10px' : '15px 5px')};
   background-color: ${props => (props.overlay ? `rgba(255, 255, 255, 0.5)` : props.theme.colors.backgroundAccentColor)};
   box-shadow: 0 2px 3px 3px rgba(0, 0, 0, 0.1);
 
-  ${props =>
-    props.overlay &&
-    css`
-      color: rgba(0, 0, 0, 0.75);
-    `}
+  ${props => (props.overlay ? 'color: rgba(0, 0, 0, 0.75);' : '')}
   & > * {
     @mixin remove-a;
-    margin: ${props => (props.overlay ? 0 : `5px`)};
```

Comment:
remove the `@mixin remove-a` this results errors. I even dunno where this mixin should be its never defined (optional)